node/Cargo.toml

```
@@ -36,6 +36,10 @@ sp-timestamp = { branch = "moonbeam-polkadot-v0.9.19", git = "https://github.com
 sp-transaction-pool = { branch = "moonbeam-polkadot-v0.9.19", git = "https://github.com/purestake/substrate" }
 substrate-frame-rpc-system = { branch = "moonbeam-polkadot-v0.9.19", git = "https://github.com/purestake/substrate" }
 
+# Try-Runtime
+
+try-runtime-cli = { branch = "moonbeam-polkadot-v0.9.19", optional = true, git = "https://github.com/purestake/substrate"}
```

Would be good if we could adhere to the general style in this file (lexicographical ordering in any case)
```suggestion
try-runtime-cli = { branch = "moonbeam-polkadot-v0.9.19", git = "https://github.com/purestake/substrate", optional = true}
```